scripts/requirements-extras.txt

```diff
@@ -15,6 +15,9 @@ junit2html
 # Script used to build firmware images for NXP LPC MCUs.
 lpc_checksum
 
+# used by NXP platform to generate/flash firmware images
+spsdk == 2.3.0
```

Comment:
Let's upversion spsdk to v2.6.0 in which a set of new parameter of nxpuuu are added, use these parameter to specify which board will be used in case of multiple boards are connected to the same host, this is important for testing.

New nxpuuu parameters provided in 2.6.o is: -up (--usbpath) or -us (--usbserial)

Reply:
okay, updated the verision. But I didn't find the parameters from the command or latest doc. Could you share?

Reply:
If you have already installed an older version of spsdk,  you need to remove the older one and re-install the latest one:
```
python -m pip install --upgrade pip
pip uninstall spsdk
pip install spsdk
```
Then use "`spsdk --version`" to make sure 2.6.0 is install.
So that you could use "nxpuuu --help" to check that the following parameters are available:
```
  -up, --usbpath USB_PATH         Filter UUU devices by USB path
  -us, --usbserial SERIAL_NUMBER  Filter UUU devices by USB serial number
```